dew/git.py

```diff
@@ -44,7 +44,7 @@ def checkout(repo: git.Repo, origin: git.Remote, head_name: str, ref: str) -> No
             with submodule.config_writer() as writer:
                 writer.set('url', actual_url)
 
-        print(f'Processing submodule {submodule.name}')
+        print(f'Processing submodule {submodule.name}', flush=True)

```

Comment:
I think this print was here when I was doing some submodule debugging. Would be cool to actually plumb in the view object that handles logging like this.